NEMP/mods.json

```diff
@@ -3723,15 +3714,159 @@
             "name": "cooking-for-blockheads"
         }
     },
+    "AdobeBlocks": {
+        "function": "CheckCurse",
+        "active": true,
+        "curse": {
+            "regex": "AdobeBlocks(?:-Forge)?(?:-(?P<mc>.+?))?-(?P<version>[0-9]+(?:\.[0-9]+)+)",
```

Comment:
Those instances of `\.` (and the ones in the other definitions) need to be changed to `\\.` so the backslash is properly escaped inside the JSON